File: practice/exercism/go/isogram/isogram.go
// Package isogram determines if strings are isograms.
package isogram

import "strings"

// IsIsogram determines if a string is an isogram.
func IsIsogram(s string) bool {

	s = strings.ToLower(s)

	for r := 'a'; r <= 'z'; r++ {
		if strings.Count(s, string(r)) > 1 {
			return false
		}
	}

	return true
}

File: practice/exercism/kotlin/isogram/src/main/kotlin/Isogram.kt
object Isogram {
    fun isIsogram(str: String): Boolean {
        var charCount = mutableMapOf<Char, Int>()
        for (ch in str.toLowerCase()) {
            var c = charCount.getOrDefault(ch, 0)
            if (c > 0 && ch != ' ' && ch != '-') return false
            charCount.put(ch, c + 1)
        }
        return true
    }
}